functions/influxdb+grafana.bash

```diff
@@ -29,15 +29,26 @@ influxdb_grafana_setup() {
   local password_check
   local text_grafana_admin_password
   local text_openHAB_integration
+  local totalmemory
+  local lowmemory
 
   FAILED=0
-  text_intro="This will install and configure InfluxDB and Grafana. For more information please consult this discussion thread:\\nhttps://community.openhab.org/t/13761/1"
+  text_intro="This will install and configure InfluxDB and Grafana. For more information please consult this discussion thread:\\nhttps://community.openhab.org/t/13761/1\\n\\nNOTE for existing installations:\\n - Grafana password will be reset and configuration adpated\\n - If local installation of InfluxDB is choosen, passwords will be reset and config files will be changed"
+  text_lowmem="WARNING: InfluxDB and Grafana tend to use a lot of memory. Your machine reports less than 1 GB memory, and we STRICTLY RECOMMEND NOT TO PROCEED!\\n\\nDISCLAIMER: Proceed on your own risk, do NOT report tickets if you run in problems."
```

Comment:
Proceed _at_ your own risk _and_ do NOT ....

Reply:
ok